src/shared/ui/badge/badge.stories.tsx

```
@@ -0,0 +1,84 @@
+  args: { variant: "secondary", children: "Secondary" },
+};
+
+export const Success: Story = {
+  args: { variant: "success", children: "Success" },
+};
+
+export const Warning: Story = {
+  args: { variant: "warning", children: "Warning" },
+};
+
+export const Error: Story = {
```

Biome (2.4.6)

[error] Do not shadow the global "Error" property.

(lint/suspicious/noShadowRestrictedNames)